### PR TITLE
Array#prepend specs

### PR DESCRIPTION
**Pull request by stomar**
(New alias in 2.5)

I had no idea before how to write specs for aliases, so I hope I figured it out correctly.

### CODE REVIEW

**Comment by stomar** [comment]
The CI failure seems unrelated.

File: core/array/shared/unshift.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    a.shift
+    a.shift
+    a.shift
+    a.send(@method, 3, 4)
+    a.should == [3, 4]
+  end
+
+  it "quietly ignores unshifting nothing" do
+    [].send(@method).should == []
+    [].send(@method, *[]).should == []
```
Comment (eregon) [inline core/array/shared/unshift.rb]: RuboCop complains about this line.
I think we should remove it, as it is equivalent to the line above and semantics of splat with an empty array are specified under `language/`.

Comment (eregon) [reply core/array/shared/unshift.rb]: Could you remove that line?

**Comment by stomar** [comment]
Done.

**Pull request merged**